src/components/SideNav/SideNavMethods.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { screen } from '@testing-library/react'
+import { pick } from 'lodash'
+import { withThemeProvider } from '@looker/components-test-utils'
+
+import { api } from '../../test-data'
+import { renderWithSearchAndRouter } from '../../test-utils'
+import { SideNavMethods } from './SideNavMethods'
+
+describe('SideNavMethods', () => {
+  const tag = 'Dashboard'
+  const methods = api.tags[tag]
+
+  test('it renders provided methods', () => {
+    renderWithSearchAndRouter(
+      withThemeProvider(
+        <SideNavMethods methods={methods} tag={tag} specKey={'3.1'} />
+      )
+    )
+    const sideNavItems = screen.getAllByRole('link')
+    expect(sideNavItems.length).toEqual(Object.keys(methods).length)
+    expect(sideNavItems[0].closest('a')).toHaveAttribute(
```

Comment:
I think `.closest('a')` is not required since the `sideNavItems[0]` is itself an `a`:

```suggestion
    expect(sideNavItems[0]).toHaveAttribute(
```